spring-cloud-stream-app-generator/pom.xml

```diff
@@ -43,6 +43,10 @@
 						<aggregate-counter-sink>
 							<extraTestConfigClass>org.springframework.cloud.stream.app.test.aggregate.counter.AggregateCounterSinkTestConfiguration.class</extraTestConfigClass>
 						</aggregate-counter-sink>
+						<azure-blob-sink>
+							<autoConfigClass>spring.cloud.starter.stream.sink.azure.blob.class</autoConfigClass>
```

Comment:
You missed `AzureBlobSinkConfiguration` class name here. 
Although, I think we don't need to specify this option: `azure-blob-sink` is mapped to the `AzureBlobSinkConfiguration` pretty straightforward.